app/src/main/java/org/wikipedia/travel/destinationpicker/DestinationFragment.java

```diff
@@ -18,32 +24,36 @@
 
 import org.wikipedia.R;
 import org.wikipedia.activity.FragmentUtil;
+import org.wikipedia.travel.trip.Trip;
+
+import java.util.ArrayList;
+import java.util.List;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.Unbinder;
 
+import static android.widget.LinearLayout.VERTICAL;
+
 /**
  * Created by abhandal on 3/3/2018.
  */
 
 public class DestinationFragment extends Fragment {
 
+    @BindView(R.id.destination_history_view_recycler)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)